--- "f:\\RadMix\\zlib\\CMakeLists.txt"	2014-07-25 05:17:22.099961900 -0500
+++ CMakeLists.txt	2015-06-06 22:15:46.169713200 -0500
@@ -5,6 +5,9 @@
 
 set(VERSION "1.2.8")
 
+set(CMAKE_C_FLAGS_RELEASE "${CMAKE_C_FLAGS_RELEASE} /MT")
+set(CMAKE_C_FLAGS_DEBUG "${CMAKE_C_FLAGS_DEBUG} /MTd")
+
 option(ASM686 "Enable building i686 assembly implementation")
 option(AMD64 "Enable building amd64 assembly implementation")
 
